src/sagemaker/mxnet/README.rst

```diff
@@ -31,13 +31,68 @@ In the following sections, we'll discuss how to prepare a training script for ex
 Preparing the MXNet training script
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-+-------------------------------------------------------------------------------------------------------------------------------+
-| WARNING                                                                                                                       |
-+===============================================================================================================================+
-| This required structure for training scripts will be deprecated with the next major release of MXNet images.                  |
-| The ``train`` function will no longer be required; instead the training script must be able to be run as a standalone script. |
-| For more information, see `"Updating your MXNet training script" <#updating-your-mxnet-training-script>`__.                   |
-+-------------------------------------------------------------------------------------------------------------------------------+
++--------------------------------------------------------------------------------------------------------------------------------------------------+
+| WARNING                                                                                                                                          |
++==================================================================================================================================================+
+| The structure for training scripts changed with MXNet version 1.3.                                                                               |
+| Please make sure you are referring to the correct section of this README when preparing your script.                                             |
+| To jump to how to upgrade an old script to the new format, see `"Updating your MXNet training script" <#updating-your-mxnet-training-script>`__. |
++--------------------------------------------------------------------------------------------------------------------------------------------------+
+
+For versions 1.3 and higher
+'''''''''''''''''''''''''''
+Your MXNet training script must be a Python 2.7 or 3.5 compatible source file.
+
+The training script is very similar to a training script you might run outside of SageMaker, but you can access useful properties about the training environment through various environment variables, such as
```

Comment:
"...environment variables, including the following:"

Question: Is this an exhaustive list of the available environment variables? Does one exist?